src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
```

Comment:
I think it's easier to destruct **person** here to avoid confusion and write less code!

Reply:
> I think it's easier to destruct **person** here to avoid confusion and write less code!

Make sense, but it's not so important in this case, you can choose